### PR TITLE
move to vsc-install

### PR DESCRIPTION
**Pull request by stdweird**

### CODE REVIEW

**Comment by stdweird** [comment]
test this please

**Comment by JensTimmerman** [comment]
jenkins: test this please, paramiko was installed under python2.7

File: setup.py [hunk 1]
```diff
@@ -30,36 +30,16 @@
...
     'version': '1.7.4',
     'author': [jt],
     'maintainer': [jt],
-    'packages': ['vsc', 'vsc.manage'],
-    'namespace_packages': ['vsc'],
-    'scripts': ['bin/misty.py'],
     'data_files': [('/etc', ['config/manage_defaults.cfg'])],
     'install_requires': [
+        'vsc-base >= 2.4.16',
         'paramiko',
```
Comment (JensTimmerman) [inline setup.py]: I guess paramiko should be  `requires`, and not `install_requires`?

**Comment by JensTimmerman** [comment]
for some reason this python2.7 picks up the system python 2.7 libs, including the Crypto one, which is built with different unicode support: 
http://effbot.org/pyfaq/when-importing-module-x-why-do-i-get-undefined-symbol-pyunicodeucs2.htm

File: setup.py (hunk 1 above)
Comment (stdweird) [inline setup.py]: there is no `requires`? but there is a `setup_requires`

File: bin/misty.py [hunk 2]
```diff
@@ -76,5 +75,5 @@ def main():
     #(which nodes differ, and those with errors)
     #done
 
-
-main()
+if __name__ == '__main__':
+    main()
```
Comment (JensTimmerman) [inline bin/misty.py]: why is this needed? This is a bin file, you shouldn't try to import it.
